src/main/proto/com/vinaacademy/grpc/user_service.proto

```
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package com.vinaacademy.grpc;
```

Buf (1.57.2)

Files with package "com.vinaacademy.grpc" must be within a directory "com/vinaacademy/grpc" relative to root but were in directory "src/main/proto/com/vinaacademy/grpc".

(PACKAGE_DIRECTORY_MATCH)